<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1503</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>POSICION EN Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Emoji</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>SizeHorCursor</cursorShape>
    </property>
    <property name="text">
     <string>Registro Neuronas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GREEN">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Principio">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Al principio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_ID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="POSICIONY">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_VOLTAJE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RED">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>POSICION EN X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>VOLTAJE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Final">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Al final</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Muestra">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>muestra</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="POSICIONX">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BLUE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="RESULTADO">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Guardar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Recuperar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Recuperar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buscar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="RESULTADO2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>721</width>
      <height>351</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Voltaje</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Posicion en X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Posicion en Y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Red</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Green</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Blue</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="busqueda">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>460</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1503</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
